clients/config-center-client/src/main/java/org/apache/servicecomb/config/center/client/AddressManager.java

```diff
@@ -17,25 +17,32 @@
 
 package org.apache.servicecomb.config.center.client;
 
-import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
 
 import org.apache.commons.lang.StringUtils;
+import org.apache.servicecomb.http.client.common.EndpointAddress;
 import org.apache.servicecomb.http.client.common.HttpUtils;
+import org.apache.servicecomb.http.client.event.CommonEventManager;
+import org.apache.servicecomb.http.client.event.RefreshEndpointEvent;
+
+import com.google.common.eventbus.Subscribe;
 
 public class AddressManager {
+
   public static final String DEFAULT_PROJECT = "default";
 
   private final String projectName;
 
-  private final List<String> addresses;
+  private EndpointAddress endpointAddress;
```

Comment:
Maybe do not need class `EndpointAddress`, the logic can put in `AddressManager`. 
Can create new `AbstractAddressManager` and all `AddressManager` extends it.